retry_test.go

```diff
@@ -533,18 +532,81 @@ func TestIsRetryable(t *testing.T) {
 	}
 }
 
-func TestExponentialJitterBackoff(t *testing.T) {
-	retryTimes := make([]float64, 10)
-	inputTime := 1.0
-	for i := 0; i < 10; i++ {
-		resultTime := defaultWaitAlgo.calculateWaitBeforeRetry(i+1, inputTime)
-		retryTimes[i] = resultTime
-		inputTime = resultTime
+func TestCalculateRetryWait(t *testing.T) {
+
+	tcs := []struct {
+		attempt      int
+		currWaitTime float64
+		minSleepTime float64
+		maxSleepTime float64
+	}{ // test for randomly selected attempt and currWaitTime values
```

Comment:
Maybe the comment to line 536 - this one is blank and awaits for some content ;)